packages/yourloops/test/components/dashboard-widgets/medical-files/weekly-report-list.test.tsx

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
There is no test about the conditionnal rendering defined by the variable `displayWeeklyReportDetails`